sql/moz-fx-data-shared-prod/braze_derived/newsletters_v1/checks.sql

```diff
@@ -10,10 +10,10 @@ ASSERT(
     `moz-fx-data-shared-prod.braze_derived.newsletters_v1`,
     UNNEST(newsletters) AS newsletters
   WHERE
-    newsletters.update_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 15 HOUR)
+    newsletters.update_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 8 HOUR)
 ) > 0
 AS
-  "No new records in the braze_derived.newsletters_v1 table in the last 15 hours";
+  "No new records in the braze_derived.newsletters_v1 table in the last 8 hours";
```

Comment:
should we make it 7 hours? otherwise it might be from the last run? probably not but in cas